scripts/binary_release.sh

```diff
@@ -53,6 +53,7 @@ function upload_binary {
 
   else
     folder="${release%-release}"
+    aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os}/${BRANCH}/${folder}/$1
     aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os}/${BRANCH}/${COMMIT}/${folder}/$1
     aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os_and_arch}/${BRANCH}/${COMMIT}/${folder}/$1
```

Comment:
We're already uploading metadata that should describe where the binary is. Is it really worth the convenience of not having to inspect the metadata if it means three whole copies of the same binary in the bucket?

Reply:
Currently, we configure the forknet tool with an URL and we have a command to download the binaries again. It would be helpful to have a constant release path. 
Custom release is currently on demand. Is there another place where we use this?
Is there a way to create a link in S3 instead of uploading the files?

Reply:
From what I see, S3 does now allow links but the binaries that we generate are ~10MB. I Believe that we can go ahead with it since it is only on demand.